website/docusaurus.config.ts

```diff
@@ -104,6 +104,12 @@ const config: Config = {
                     label: "Blog",
                     position: "left",
                 },
+                {
```

Comment:
I would probably remove this and let it be indexed as another doc (inside the versioned_docs/verion-v1.x) folder

Reply:
I've considered your suggestion on integrating the MACI 1.0 Spec into our versioned documentation. However, I think maintaining it as a standalone link in the navigation could be more beneficial for users. This approach, often adopted in well-established projects, enhances the spec's visibility and accessibility, particularly for those seeking detailed technical insights. It simplifies finding and referencing this key document.

What are your thoughts on this?

Reply:
> I think maintaining it as a standalone link in the navigation could be more beneficial for users

I don't see this as necessary. It's a highly technical document & it's specific to MACI v1.0. I agree with @ctrlc03 that it makes sense within the v1 docs.

Reply:
Fair enough

Reply:
Reflected in 2d1e3f2